src/PSRule.Rules.Azure/rules/Azure.APIM.Rule.ps1

```diff
@@ -310,6 +310,26 @@ Rule 'Azure.APIM.CORSPolicy' -Ref 'AZR-000365' -Type 'Microsoft.ApiManagement/se
     }
 }
 
+# Synopsis: Base element for any policy element in a section should be configured.
+Rule 'Azure.APIM.PolicyBase' -Ref 'AZR-000371' -Type 'Microsoft.ApiManagement/service', 'Microsoft.ApiManagement/service/apis', 'Microsoft.ApiManagement/service/apis/resolvers', 'Microsoft.ApiManagement/service/apis/operations', 'Microsoft.ApiManagement/service/apis/resolvers/policies', 'Microsoft.ApiManagement/service/products/policies', 'Microsoft.ApiManagement/service/apis/policies',
+'Microsoft.ApiManagement/service/apis/operations/policies' -If { $Null -ne (GetAPIMPolicyNode -Node 'policies' -IgnoreGlobal) } -Tag @{ release = 'GA'; ruleSet = '2023_06'; } {
+    $policies = GetAPIMPolicyNode -Node 'policies'
+    foreach ($policy in $policies) {
+        Write-Debug "Got policy: $($policy.OuterXml)"
+        
+        $inboundSection = @($policy.inbound)
+        $Assert.HasField($inboundSection, 'base')
+
+        $backendSection = @($policy.backend)
+        $Assert.HasField($backendSection, 'base')
+
+        $outboundSection = @($policy.outbound)
+        $Assert.HasField($outboundSection, 'base')
+
+        $onerrorSection = @($policy.'on-error')
+        $Assert.HasField($onerrorSection, 'base')
```

Comment:
Ah yes, I think we need to approach it slightly differently. Because `$policy.inbound` is an array we can't use `HasField` directly. Try this:

```suggestion
        $Assert.HasField($policy, 'inbound.base')
        $Assert.HasField($policy, 'backend.base')
        $Assert.HasField($policy, 'outbound.base')
        $Assert.HasField($policy, 'on-error.base')
```